Perlang.Tests.Architecture/Perlang.Tests.Architecture.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+      <DocumentationFile>bin\Debug\Perlang.Tests.Architecture.xml</DocumentationFile>
+        <NoWarn>SA1300;1591</NoWarn>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+      <DocumentationFile>bin\Release\Perlang.Tests.Architecture.xml</DocumentationFile>
+      <NoWarn>SA1300;1591</NoWarn>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
```

Comment:
Worth pointing out is that when creating the new project, I got the 16.9.1 version at first. When I tried bumping all the references to that package to the newer version, I got loads and loads of errors like this:

```
  Error Message:
   System.Reflection.ReflectionTypeLoadException : Unable to load one or more of the requested types.
Could not load file or assembly 'Microsoft.VisualStudio.Coverage.CoreLib.Net, Version=16.9.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.

  Stack Trace:
     at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   at System.Reflection.RuntimeModule.GetTypes()
   at System.Reflection.Assembly.GetTypes()
   at Perlang.Interpreter.PerlangInterpreter.<>c.<RegisterGlobalClasses>b__15_0(Assembly a) in /home/per/git/perlang/Perlang.Interpreter/PerlangInterpreter.cs:line 124
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at Perlang.Interpreter.PerlangInterpreter.RegisterGlobalClasses() in /home/per/git/perlang/Perlang.Interpreter/PerlangInterpreter.cs:line 132
   at Perlang.Interpreter.PerlangInterpreter.RegisterGlobalFunctionsAndClasses() in /home/per/git/perlang/Perlang.Interpreter/PerlangInterpreter.cs:line 110
   at Perlang.Interpreter.PerlangInterpreter..ctor(Action`1 runtimeErrorHandler, Action`1 standardOutputHandler, IEnumerable`1 arguments, Boolean replMode) in /home/per/git/perlang/Perlang.Interpreter/PerlangInterpreter.cs:line 83
   at Perlang.Tests.Integration.EvalHelper.EvalWithParseErrorCatch(String source) in /home/per/git/perlang/Perlang.Tests.Integration/EvalHelper.cs:line 77
   at Perlang.Tests.Integration.Var.VarTests.use_false_as_var() in /home/per/git/perlang/Perlang.Tests.Integration/Var/VarTests.cs:line 368
```

I don't really understand why, and it feels less important to dig into this _right now_. Noting it down for future reference though.